storage/badger/procedure/index.go

```diff
@@ -35,10 +39,17 @@ func RetrieveIndex(blockID flow.Identifier, index *flow.Index) func(tx *badger.T
 		if err != nil {
 			return fmt.Errorf("could not retrieve seal index: %w", err)
 		}
+		var receiptIDs []flow.Identifier
+		err = operation.LookupPayloadReceipts(blockID, &receiptIDs)(tx)
+		if err != nil {
+			return fmt.Errorf("could not retrieve receipts index: %w", err)
+		}
 
 		*index = flow.Index{
+			NodeIDs:       nil,
```

Comment:
⚠️ 
The field  https://github.com/onflow/flow-go/blob/2375004eb5ecee3a16889403b1b75eea0832ee2b/model/flow/index.go#L4  which we neither persist to storage nor populate when retrieving the struct. This field is essentially subject to the same bug you just fixed (if it was ever used).  I believe it is not used in our code base (an artifact from the time we didn't have the protocol state). 
Would you mind removing `NodeIDs` as part of your PR? Thanks.